blocks/test/fixtures/core__audio.json

```diff
@@ -5,8 +5,9 @@
         "isValid": true,
         "attributes": {
             "src": "https://media.simplecast.com/episodes/audio/80564/draft-podcast-51-livePublish2.mp3",
-            "align": "right"
+            "align": "right",
+		  	"caption": []
```

Comment:
The indentation is weird because the file is semi-generated, we should be consistent with the generated output. (use spaces)